.github/workflows/cron-storage-limits.yaml

```
@@ -0,0 +1,45 @@
+name: Storage Limit Email Notifications
+
+on:
+  schedule:
+    - cron: '0 5 * * *'
+  # Including 'workflow_dispatch' here allows the job to be triggered manually,
+  # as well as on the schedule.
+  workflow_dispatch:
+
+jobs:
+  send-notifications:
+    name: Send notifications
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO: Update this to include production once it's been successfully
+        # tested on staging
+        # env: ['staging', 'production']
+        env: ['staging']
+    timeout-minutes: 100
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
```

for consistency with our other workflows...
```suggestion
        uses: actions/setup-node@v2
```